virttest/qemu_devices/qdevices.py

```diff
@@ -1618,7 +1618,16 @@ def _cmdline_json(self):
         params = self.params.copy()
         self._convert_memobj_args(params)
         command_dict["qom-type"] = params.pop("backend")
-        return out + '\'' + json.dumps(dict(command_dict, **params)) + '\''
+        for key, val in params.items():
+            # prealloc, dump, merge
+            # from -object ("qom-type": "memory-backend-ram")
+            if key in ("prealloc", "dump", "merge"):
+                command_dict[key] = val.lower() in ("yes", "on")
+            elif key == "host-nodes":
+                command_dict[key] = list(map(int, val.split()))
+            else:
+                command_dict[key] = val
```

Comment:
hi @nickzhq , do you think this part of changes should be moved to _convert_memobj_args function? which is used for converting values for json format, cc @YongxueHong